content/en/docs/setup/install/virtual-machine/index.md

```diff
@@ -32,64 +27,33 @@ but not production. Like all alpha features, this guide is subject to change.
 
 ## Prepare the guide environment
 
-1. Set the environment variables `"${ISTIO_DIR}"`, `"${WORK_DIR}"`, your cluster
-    name, and the service namespace. Ensure `"${WORK_DIR}"` is prefixed with `"${HOME}"`
+1. Create a virtual machine
+1. Set the environment variables `"${VM_NAME}"`, `"${WORK_DIR}"`,`"${CLUSTER_NAME}"` , `"${SERVICE_ACCOUNT}"`,
+，`"${SERVICE_ACCOUNT}"` your cluster name, and the service namespace. Ensure `"${WORK_DIR}"` is prefixed with `"${HOME}"`
     (e.g. `WORK_DIR="${HOME}"/vmintegration`).
 
     {{< text bash >}}
-    $ ISTIO_DIR="<the directory containing an unarchived version of Istio>"
+    $ VM_NAME="<the name of your vm instance you created>"
     $ CLUSTER_NAME="<the name of your cluster>"
     $ SERVICE_NAMESPACE="<the name of your service namespace>"
```

Comment:
SERVICE_NAMESPACE -> VM_NAMESPACE

the namespace that you want to place your VM at.

Reply:
OK will change it